duckduckgo/Android
# Add autofill default state param to key autofill pixels

Task/Issue URL: https://app.asana.com/0/0/1205431733474503/f 

### Description
Adds an additional param to certain autofill pixels which denote whether autofill was enabled by default or not. Note, this is not the same as whether autofill is currently enabled. 

The logic to determine if enabled by default is downstack of this one; this PR pertains only to attaching the pixel params.

### Steps to test this PR

💡 To make this easier to verify:
- add a breakpoint to `AutofillPixelIntereptor line 69` and debug it
- this will break only when it's found a pixel it needs to add params to

#### Play build

- [x] Fresh install `play` build  (this should mean `default_state=off` is the param that is added)
- [x] Visit `overflow->logins`. Verify breakpoint hit, and param added for `m_nav_autofill_menu_item_pressed`
- [x] Visit `overflow->settings-> logins`. Verify breakpoint hit, and param added for `m_autofill_settings_opened` 
- [x] Enable the toggle for `Save and Autofill Logins`; verify breakpoint hit and param added with `default_state=off`
- [x] Disable the toggle for `Save and Autofill Logins`; verify breakpoint hit and param added with `default_state=off`
- [x] Trigger a few pixels which have nothing to do with the autofill interceptor pixels, and verify they **do not** get the `default_state` param added (i.e., the breakpoint isn't hit)

#### Internal build
- [x] Fresh install `internal` build (this should mean `default_state=on` for the added param)
- [x] Visit `overflow->logins`. Verify breakpoint hit, param added for `m_nav_autofill_menu_item_pressed` and is `default_state=on`
